### DISCUSSION TITLE
[委托][整活]这个组织是做什么的？

### DISCUSSION THREAD

**Discussion by SeeleVollereid** [share121/inter-knot#4554] [崩坏3]
我在街上走着走着就遇上了这个组织的一位绿色头发的人，她拉着我说让我加入这个组织，我当时不知道这个组织是干什么的，所以我拒绝了，但是我想知道这个组织是做什么的。logo如图
![1725713444183](https://github.com/user-attachments/assets/75cffcfe-12e6-430c-8a5e-98295489ed1c)
![img_-688530291_1719056039893](https://github.com/user-attachments/assets/e02f0e2e-94b3-4503-ba44-56cda1f7f8af)

**Comment by vinori233** [comment]
是为了世上的爱与美好而战ヾﾉ≧∀≦)o

**Comment by Allium0** [comment]
是美少女的魔法♪

**Comment by lmingzhe86** [comment]
敢死大队！

**Reply by kunga555** [reply]
神tm敢死大队
